modules/core/src/lib/layer.ts

```diff
@@ -547,7 +547,7 @@ export default abstract class Layer<PropsT extends {} = {}> extends Component<
   /** (Internal) Propagate an error event through the system */
   raiseError(error: Error, message: string): void {
     if (message) {
-      error.message = `${message}: ${error.message}`;
+      error = new Error(`${message}: ${error.message}`, {cause: error});
```

Comment:
Aren't we duplicating information by both slapping message of the source error into the message of the derived error and then attaching the source error as cause?

Reply:
Do we need a fallback if Error constructor throws on older environment that doesn't support options? Or are options just ignored? Or we just don't support?